azkaban-web-server/src/main/java/azkaban/webapp/servlet/WebUtils.java

```diff
@@ -93,6 +93,8 @@ public static String formatStatus(final Status status) {
         return "Killing";
       case DISPATCHING:
         return "Dispatching";
+      case EXECUTION_STOPPED:
+        return "Execution stopped, crashed executor/container";
```

Comment:
How is this message used? Can we remove the "crashed executor/container" part which is an implementation detail?